src/core/public/chrome/ui/header/collapsible_nav.tsx

```diff
@@ -103,10 +101,6 @@ interface Props {
   navigateToUrl: InternalApplicationStart['navigateToUrl'];
   customNavLink$: Rx.Observable<ChromeNavLink | undefined>;
```

Comment:
The `customNavLink$` is not used anymore. Do we need to add its logic back?

Reply:
Maybe we need this props in the future. What about just keep it currently?